### PR TITLE
Refactor/favicon input

### PR DESCRIPTION
**Pull request by adon-cloudcannon**
# Context

[Link to Notion ticket](https://www.notion.so/cloudcannon/Favicon-input-0eab4bae6b704470b2fe15e2509d5683)

# Additional information

- Check input works as image input
- Confirm favicon loads? Image upload works, but favicon not loaded yet - but maybe this takes a little while to come through?

# Testing (tester)

The reviewer should check on this branch:

- The code
- The component in Bookshop browser
- The site in CloudCannon [(link here)](https://app.cloudcannon.com/42158/editor#sites/119944/dashboard/summary:/edit?editor=visual&url=%2F&collection=pages&path=%2Fsrc%2Fpages%2Findex.html)

# Before merge (PR owner)

- [ ] Delete the test site in CloudCannon
- [ ] For "generic" components **only**: remove `- content_blocks` from `structures` in the YAML file (this was added for testing).

### CODE REVIEW

**Review by calumocannon — CHANGES_REQUESTED** [review]
The path for the favicon is pointing at the logo png instead of the favicon svg. I found I was getting a 404 for both the images in the current path so I moved them out of `uploads`

I made a PR in to this one which you can feel free to accept or ignore. It looks scarier than it is due to also merging in main. PR #99

**Comment by calumocannon** [comment]
Also we should remove the old favicon (Prima Mesa P)

**Comment by adon-cloudcannon** [comment]
The issue was:

- awhile ago our uploads folder was only src/uploads
- that didn't work for some other scenario, so it was changed to src/assets/uploads
- but the .eleventy.js file was never updated at that same time for the files passthrough
- updated now and the favicon input seems to work 1000000%

File: src/_data/seo.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-favicon: /assets/images/logos/favicon.png
+favicon: /assets/uploads/nuxtjs.svg
 title: Primera Mesa
```
Comment (calumocannon) [inline src/_data/seo.yml:2]:
```suggestion
title: Venture
```
Cheeky perhaps but should we update this while we are here ?

**Review by calumocannon — APPROVED** [review]
Very nice 👏 Thanks for the fix.

Approved assuming the testing changes get reverted.

**Pull request merged**